ci-operator/config/openshift/release/openshift-release-master__nightly-4.15.yaml

```diff
@@ -1040,26 +1040,177 @@ tests:
       BASE_DOMAIN: origin-ci-int-aws.dev.rhcloud.com
       FEATURE_SET: TechPreviewNoUpgrade
     workflow: openshift-e2e-aws-shared-vpc-phz
-- as: e2e-agent-compact-ipv4
+- as: e2e-agent-compact-ipv4-conformance
   cluster: build05
   cron: 0 */12 * * *
   steps:
     cluster_profile: equinix-ocp-metal
-    workflow: agent-e2e-compact-ipv4
+    env:
+      DEVSCRIPTS_CONFIG: |
+        AGENT_E2E_TEST_SCENARIO=COMPACT_IPV4
+        FIPS_MODE=true
+      TEST_SKIPS: \[sig-auth\]\[Feature:HTPasswdAuth\] HTPasswd IDP should successfully
```

Comment:
Couldn't be this one defined directly in `agent-e2e-generic-conformance` workflow? In such way not only this file will be less noisy, but it will be possible for our team to address the skipped tests easily (no changes in this repo)

Reply:
(btw, my expectation would be, at this level, apart the `DEVSCRIPTS_CONFIG`, just the `TEST_SUITE` defined)

Reply:
Some tests were only failing for one cluster topology so kept it seperate that way it will be easier to identify which ones are failing for a particular cluster setting and debug the failures per cluster topology.

Reply:
I wouldn't recommend CI for debugging failures, in any case a PR can be always opened with a limited set of tests, just for the sake of troubleshooting